v2_papers/2010/ITC_Demystified.json

```diff
@@ -0,0 +1,289 @@
+{
+  "version": "v2.1",
+  "reference": {
+    "title": "Internet Traffic Classification Demystified: On the Sources of the Discriminative Power",
+    "authors": [
+      "Yeon-sup Lim",
+      "Hyun-chul Kim",
+      "Jiwoong Jeong",
+      "Chong-kwon Kim",
+      "Ted “Taekyoung\" Kwon",
+      "Yanghee Choi"
+    ],
+    "publication_name": "Co-NEXT '10 Proceedings of the 6th International COnference",
+    "publication_type": "peer_reviewed_conference",
+    "year": 2010,
+    "organization_publishers": [
+      "acm"
+    ],
+    "pages_number_of": 12,
+    "bibtex": {
+      "type": "",
+      "volume": "",
+      "issue": "",
+      "pages": ""
```

Comment:
(same thing as above)
These bibtex fields must not be empty strings. If there's no content for them, remove them.
"type" field is mandatory though.
For the bibtex fields I recommend copying the values from the bibtex provided by google scholar (google for the paper in scholar, and look at the citation button).